src/components/GrampsjsFormEditNoteType.js

```diff
@@ -0,0 +1,35 @@
+/*
+Form for adding a new event reference
+*/
+
+import {html} from 'lit'
+
+import './GrampsjsFormSelectType.js'
+
+import {GrampsjsObjectForm} from './GrampsjsObjectForm.js'
+
+class GrampsjsFormEditNoteType extends GrampsjsObjectForm {
+  renderForm() {
+    return html`
+      <grampsjs-form-select-type
+        id="note-type"
+        .strings="${this.strings}"
+        ?loadingTypes="${this.loadingTypes}"
+        ?disabled="${this.loadingTypes}"
```

Comment:
I've removed `disabled` property, as it was not used internally in `grampsjs-form-select-type`. In all cases it had the same value as `loadingTypes` so in updated version the component uses it directly in order to disable select box

Reply:
Ok, let's merge this after #141.